install.sh

```diff
@@ -40,10 +50,40 @@ function check_mvn() {
     fi
 }
 
+function clone_rep() {
+    sut_dir=$1
+    rep_url=$2
+    rep_com=$3
+    echo "Cloning repository $rep_url commit $rep_com to $sut_dir"
+    git clone $rep_url $sut_dir
+    if [[ -n "$rep_com" ]]; then
+        ( cd $sut_dir ; git checkout $rep_com ) #; rm -rf $sut_dir/.git )
+    fi
+}
+
+function install_protocolstatefuzzer() {
+    echo $PROTOCOLSTATEFUZZER_FOLDER
+    if [[ -d $PROTOCOLSTATEFUZZER_FOLDER ]]; then
+        echo "$PROTOCOLSTATEFUZZER_FOLDER folder already exists"
+        echo "Skipping ProtocolState-Fuzzer setup"
+    else
+        clone_rep $PROTOCOLSTATEFUZZER_FOLDER $PROTOCOLSTATEFUZZER_REP_URL $PROTOCOLSTATEFUZZER_COMMIT
+        (
+            cd $PROTOCOLSTATEFUZZER_FOLDER
+            echo "Patching ProtocolState-Fuzzer for compatibility with Java 11"
+            git apply $PROTOCOLSTATEFUZZER_PATCH
+            echo "Installing ProtocolState-Fuzzer"
+            mvn install
+        )
+    fi
+}
+
 # Check for the presence of Java and maven and if 'apt-get' is present, install them
 check_java
 check_mvn
 
+# Install ProtocolState-Fuzzer before patching for Java 11
```

Comment:
The comment here is misleading, I think. It should be deleted or changed.
Both patching for Java 11 and installation happen by the same function and patching takes place first.

Reply:
Fixed!